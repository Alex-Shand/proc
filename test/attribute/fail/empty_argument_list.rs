#[proc::attribute]
fn test() -> proc::TokenStream {
    todo!()
}

fn main() {}
