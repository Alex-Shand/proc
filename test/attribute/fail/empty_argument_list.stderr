error: The attribute function should have one argument
 --> test/attribute/fail/empty_argument_list.rs:2:1
  |
2 | fn test() -> proc::TokenStream {
  | ^^
